src/main/java/MorseCodePresenter.java

```diff
@@ -0,0 +1,20 @@
+public class MorseCodePresenter
+{
+    MorseCode model;
+    MorseCodeFrame frame;
+    public MorseCodePresenter(MorseCode model, MorseCodeFrame frame)
+    {
+        this.model = model;
+        this.frame = frame;
+    }
+    public void EtoM(String english)
```

Comment:
We do not need to shorten method names in Java. This can be spelled out instead of being a cryptic word.